flatgfa/src/flatgfa.rs

```diff
@@ -277,8 +283,8 @@ impl<'a> FlatGFA<'a> {
 
     /// Look up a CIGAR alignment.
     pub fn get_alignment(&self, overlap: Span<AlignOp>) -> Alignment {
-        Alignment {
-            ops: &self.alignment[overlap],
+        Alignment { 
+            ops: &self.alignment[overlap]
```

Comment:
Can you please run `cargo fmt`? It should get rid of formatting "noise" changes like this one.